src/components/InfoPanel/_info-panel.scss

```diff
@@ -34,7 +34,7 @@
       position: absolute;
       right: 10px;
 
-      @include medium {
+      @include desktop {
```

Comment:
Drive-by fix for the full-screen icon which wasn't visible on large screens (it was only visible at the medium size).